app/jobs/create_account_fcrepo_job.rb

```diff
@@ -0,0 +1,6 @@
+# This job is the same as its parent, but chains to the next step
+class CreateAccountFcrepoJob < CreateFcrepoEndpointJob
```

Comment:
This approach seems completely reasonable (especially if `CreateDefaultAdminSetJob` is idempotent). Is there a reason to subclass the other job?